interfaces/apparmor/apparmor.go

```diff
@@ -35,6 +35,17 @@ import (
 	"github.com/snapcore/snapd/osutil"
 )
 
+// ValidateNoAppArmorRegexp will check that the given string does not
+// contain AppArmor regular expressions (AARE), double quotes or \0.
+func ValidateNoAppArmorRegexp(s string) error {
+	const AARE = `?*[]{}^"` + "\x00"
```

Comment:
Curious about the peculiar syntax here, is there a reason for this concatenation?

Reply:
Because its a "`" type string I can't embedd the \0 in there easily. I could change it to:
```
	const AARE = "?*[]{}^\"\x00"
```
not sure if that is more readable though (I have no opinion either way, both look a bit hard to read to me).